src/main/java/teammates/ui/controller/AdminActivityLogPageData.java

```diff
@@ -276,15 +283,30 @@ private QueryParameters parseQuery(String query) throws Exception{
                     versions.add(values[j].replace(".", "-"));
                 }
                 
+            } else if (label.equals("from")) {
+                SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yy HH:mm");
+                Date d = sdf.parse(values[0] + " 00:00");                
+                Calendar cal = Calendar.getInstance();
+                cal.setTime(d);
+                cal = TimeHelper.convertToUserTimeZone(cal, - Const.SystemParams.ADMIN_TIMZE_ZONE_DOUBLE);
+                fromDateValue = cal.getTime().getTime();
+                
+            } else if (label.equals("to")) {
+                SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yy HH:mm");
+                Date d = sdf.parse(values[0] + " 23:59");                
+                Calendar cal = Calendar.getInstance();
```

Comment:
Now that I think about it, yes it makes sense to update this. I just copied this block of code from the prev implementation and moved it here haha